### PR TITLE
api-docs: add importname

### PR DESCRIPTION
**Pull request by pinheadmz**
Closes https://github.com/handshake-org/handshake-org.github.io/issues/70

See https://github.com/handshake-org/hsd/pull/408

### CODE REVIEW

**Comment by kilpatty** [comment]
LGTM

**Comment by tynes** [comment]
This PR doesn't include the built frontend, is this something that you plan on adding in another PR? You mentioned about splitting up the commits between changes to `.md` and builds, maybe this is something that we could document in the `README`. I agree that the changes should be in different commits.

Is your suggestion to also separate the changes into separate PRs, potentially bundling a bunch of different changes to `.md` files?

File: src/api-docs/source/includes/_wallet_rpc_auctions.md [hunk 1]
```diff
@@ -1347,3 +1347,60 @@ Name | Default |  Description
...
+been placed on a name they are interested in bidding on themselves, they may
+execute this RPC call and include a `height` parameter, which should be any block
+*before* the OPEN for the name was confirmed. The OPEN transaction *must* be included
+in the rescan or the wallet will not track BIDs on the name.
+
+Once the auction is rescanned, [`rpc getbids`](#getbids) can be used to return
+all current BIDs on a name, even if the wallet has not placed any BIDs itself.
+
+### Params
+Name | Default |  Description
```
Comment (tynes) [inline src/api-docs/source/includes/_wallet_rpc_auctions.md:1403]: These tables generally have incorrect column names.

Comment (pinheadmz) [reply src/api-docs/source/includes/_wallet_rpc_auctions.md:1403]: oh dang you're right, that would be a good follow up

**Comment by pinheadmz** [comment]
Going to merge this @kilpatty @tynes let's try to use the github Approval function instead of "LGTM". It'll help us sort approved PRs and just feels a bit more official.

**Pull request merged**